linux_os/guide/services/ftp/disabling_vsftpd/package_vsftpd_removed/rule.yml

```diff
@@ -10,7 +10,7 @@ rationale: |-
     Removing the <tt>vsftpd</tt> package decreases the risk of its
     accidental activation.
 
-severity: high
+severity: low
```

Comment:
I don't see a need to change the severity here as you can run vsftpd as a user.

Reply:
Ok, reverting.